DynmapCore/src/main/java/org/dynmap/DynmapWorld.java

```diff
@@ -128,6 +128,7 @@ public void activateZoomOutFreshen() {
     private static final int[] stepseq = { 3, 1, 2, 0 };
 
     private void processZoomFile(MapTypeState mts, MapStorageTile tile, boolean firstVariant) {
+        final long startTimestamp = System.currentTimeMillis();
```

Comment:
The timestamp should not be the current time, but instead the time of the latest tile used within this zoomout tile

Reply:
Well spotted!